docs/publishing/sections/4_analytics_portfolio_site.md

```diff
@@ -9,3 +9,105 @@ For these situations, the [Jupyter Book-based](https://jupyterbook.org/en/stable
+    * Name your `.yml` file
+    * The structure of your `.yml` file depends on the type of your analysis:
+        * If you have one parameterized notebook with **one parameter**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            notebook: ./my-analyses/my-notebook.ipynb
+            parts:
+               - caption: Introduction
+               - chapters:
+                 - params:
+                      district_parameter: 1
+                      district_title: District 1
+            ```
+         * If you have a parameterized notebook with **multiple parameters**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            notebook: ./my-analyses/my-notebook.ipynb
+            parts:
+            - chapters:
+              - caption: County Name
+                params:
+                  paramter1_county_name
+                sections:
+                - city: parameter2_city_name
+                - city: parameter2_city_name
+            ```
+         * If you have an individual notebook with **no parameters**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            parts:
+            - caption: Introduction
+            - chapters:
+              - notebook: ./my-analyses/notebook_1.ipynb
+              - notebook: ./my-analyses/notebook_2.ipynb
+            ```
+## Running the Build Command:
+**Note:** The build command must be run from the root of the repo!
+1. Navigate back to the repo data-analyses and install the portfolio requirements with
+`pip install -r portfolio/requirements.txt`
+2. Then run `python portfolio/portfolio.py build my_report --deploy`
+    * The `my_report` will be replaced by the name of your `.yml` file in [data-analyses/portfolio/sites](https://github.com/cal-itp/data-analyses/tree/main/portfolio/sites).
+    * You also have the option to specify: run `python portfolio/portfolio.py build --help` to see the following options:
+        * `--deploy / --no-deploy`
+            * deploy this component to netlify.
+        * `--prepare-only / --no-prepare-only`
+            * Pass-through flag to papermill; if true, papermill will not actually execute cells.
+        * Other specifications:
+            * `--execute-papermill / --no-execute-papermill`
+            * `--no-stderr / --no-no-stderr`
+            * `--continue-on-error / --no-continue-on-error`
+
+3. Once this runs, you can check the preview link at the bottom of the output. It should look something like:
+    * `–no-deploy`: `file:///home/jovyan/data-analyses/portfolio/my-analysis/_build/html/index.html`
+    * `–deploy`: `Website Draft URL: https://my-analysis--cal-itp-data-analyses.netlify.app`
+
+4. Add the files using `git add` and commit!
```

Comment:
We are going to start encouraging people to use git LFS very soon for parameterized notebooks, FYI.

cc @edasmalchi